### ISSUE TITLE
button-group implementation

### ISSUE THREAD

**Issue by digitalsadhu** [fabric-ds/issues#34]
Needs UU testing

**Comment by martin-storsletten** [comment]
Selected state is not accessible with screen reader, and the component itself lacks semantics that support checked/selected/pressed state. How to address this problem depends on how this component is intended to be used. How does this component differ from e.g. Tabs or radio group in terms of functionality?
